scripts/test_mkn.py

```
@@ -59,206 +66,400 @@ def run_cmd(args, timeout=30):
+
+
+def run_lock_group_suite():
+    """Delegate to the lock group integration suite runner.
+
+    Spawns run_lock_tests.py as a subprocess so its cargo-style output
+    is streamed directly to stdout.
+
+    Returns:
+        bool: True if the suite passed.
+    """
+    result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)